packages/ui/src/ui/containers/ACL/ACL-connect-helpers.tsx

```diff
@@ -64,25 +70,24 @@ const makeAclMapStateToProps = (inputIdmKind) => {
 
         const hasApprovers = getHasApprovers(state, idmKind);
         const approversFiltered = getApproversFilteredAndOrdered(state, idmKind);
-        const columnsPermissions = getAllAccessColumnsPermissionsOrderedByInheritanceAndSubject(
-            state,
-            idmKind,
-        );
-        const objectPermissions = getAllObjectPermissionsOrderedByInheritanceAndSubject(
+        const {mainPermissions, columnsPermissions} = getAllObjectPermissionsFiltered(
             state,
             idmKind,
         );
         const columnGroups = getAllColumnGroupsActual(state, idmKind);
         const userPermissions = getAllUserPermissions(state, idmKind);
 
-        const columnsFilter = getColumnsColumns(state);
+        const columnsFilter = getAclFilterColumns(state);
 
         const auditors = getNotInheritedAuditors(state, idmKind);
         const readApprovers = getNotInheritedReadApprovers(state, idmKind);
         const responsible = getNotInheritedResponsibles(state, idmKind);
 
         const nodeType = getType(state);
 
+        const allowAclMode = idmKind !== 'path';
```

Comment:
This variable is used in one place. I propose to transfer this logic
`const aclMode = idmKind !== 'path' ? undefined : getAclCurrentTab(state);`